pandas/tests/io/test_packers.py

```diff
@@ -566,15 +566,24 @@ def test_sparse_series(self):
 
         s = tm.makeStringSeries()
         s[3:5] = np.nan
-        ss = s.to_sparse()
+        # GH 26557: DEPR
+        with tm.assert_produces_warning(FutureWarning,
```

Comment:
The same for those tests, I think in general where there is a filterwarnings for Sparse, you can add one for the to_sparse methods as well